corporate/tests/test_stripe.py

```diff
@@ -695,8 +696,10 @@ def test_upgrade_by_card(self, *mocks: Mock) -> None:
             response = self.upgrade()
         [payment_intent] = PaymentIntent.objects.all()
         assert payment_intent.stripe_payment_intent_id is not None
+
+        success = self.assert_json_success(response)
```

Comment:
I think `result_dict` or `response_dict`  would probably be a better conventional variable name for these objects than `success`.

Reply:
I’m fine with `success` or `response_dict` I guess, but `result_dict` is strictly worse. (Requests have responses; computations have results. We have a lot of `result` variables of type `HttpResponse` that should be named `response`.)

Reply:
`response_dict` WFM. I don't like `success` because the object it a dictionary containing the data returned in the response -- sans headers and the like. `success_dict` would be fine too, but I think it's cleaner to have the name directly associated with `response`.

Also agreed that it would be a nice refactor to `s/result/response/` in the tests.

Reply:
The PR is updated to go with `response_dict`.